src/virtualenv/create/via_global_ref/builtin/pypy/pypy3.py

```diff
@@ -42,6 +42,11 @@ def to_lib(self, src):
     def sources(cls, interpreter):
         for src in super(PyPy3Posix, cls).sources(interpreter):
             yield src
+        # PyPy >= 3.8 supports a standard prefix installation, where older
+        # versions always used a portable/developent style installation.
+        # If this is a standard prefix installation, skip the below:
+        if interpreter.system_prefix == "/usr":
```

Comment:
Shouldn't this be a sysconfig path rather than a constant? Is it possible to add a test for this?

Reply:
> Shouldn't this be a sysconfig path rather than a constant?

Yeah, I found myself looking for something less static, but I couldn't come up with anything better.
Any sysconfig variables from the virtualenv host wouldn't be relevant here. The ideal thing to check would be the selected sysconfig scheme on the target interpreter, but that isn't available. Do you think we should make it available?

> Is it possible to add a test for this?

What kind of thing are you looking for here? I couldn't find any tests related to any similar functionality, to extend. So it seems like I'd have to build an awful amount of test infrastructure, to test a very minor behavioural change.

Reply:
Doesn't sound minor to me 🙂 the hard part in new feature additions is always the test, never the functionality change. And yes that does imply writing new test infrastructure for it.

Reply:
How's this? I had to mock `Path()` but otherwise it isn't *too* ugly.